api/src/routers/meals.js

```diff
@@ -0,0 +1,204 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const Meals = express.Router();
```

Comment:
just a slight note, uppercase naming is by convention usually reserved for classes, components and interfaces. Regular js functions and objects are named staring from lowercase (by convetion) :)